.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,24 @@
+---
+name: 📚 Documentation
+about: Are our docs missing or lacking? Need some attention that's big enough for its own ticket? Tell us more.
```

Comment:
I'm being pedantic, but "Need some attention..." is vague and may make reference to the subject rather than make reference to the object. "Do they need some attention..." might be better here.

Reply:
Pedantry addressed (FWIW, I'm all about the pedantry, see #1)